virtcontainers/sandbox.go

```diff
@@ -1923,6 +1923,14 @@ func (s *Sandbox) calculateSandboxMemory() int64 {
 
 		if m := c.Resources.Memory; m != nil && m.Limit != nil {
 			memorySandbox += *m.Limit
+			s.Logger().WithField("memory limit", memorySandbox).Info("Memory Sandbox + Memory Limit ")
+		}
+
+		//Add hugepages memory
+		//HugepageLimit is uint64 - https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go#L242
+		for _, l := range c.Resources.HugepageLimits {
+			memorySandbox += int64(l.Limit)
```

Comment:
Is the limit from the spec in bytes, or in # of (huge)pages.  If the latter, don't you need to scale this up by the hugepage size?

Reply:
Its in bytes

Reply:
@dgibson regarding your comment on pageSizeStr, I'm unable to provide inline response. Not sure why. So putting it here.
The mount options uses something like `pagesize=<pageSizeStr>` eg. `pagesize=1G`.  So using pageSizeStr to refer to the pagesize (1G, 2M etc.)